app/controllers/users_controller.rb

```diff
@@ -1,16 +1,30 @@
 class UsersController < ApplicationController
-
   def new
     @user = User.new
   end
 
   def create
-   @user = User.new(user_params)
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to user_path(@user)
+    else
+      respond_with(@user)
```

Comment:
I think you can just use:
```
@user = User.create(user_params)

respond_with @user
```

Reply:
This is fixed with the change in 1fb4972